src/app/pages/CpsMap/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import compose from 'ramda/src/compose';
+import CpsAssetPageMain from '../../containers/CpsAssetPageMain';
+import withContexts from '../../containers/PageHandlers/withContexts';
+import withPageWrapper from '../../containers/PageHandlers/withPageWrapper';
+import withError from '../../containers/PageHandlers/withError';
+import withLoading from '../../containers/PageHandlers/withLoading';
+import withData from '../../containers/PageHandlers/withData';
+
+const CpsMapContainer = props => {
+  console.log(`MAP Container`);
```

Comment:
Should we use the node logger instead of the console (or is this just temporary)?

Reply:
this was just temporary to ensure that the correct pagetype was being rendered. have removed these now.